builtin/providers/aws/opsworks_layers.go

```diff
@@ -328,6 +356,15 @@ func (lt *opsworksLayerType) Create(d *schema.ResourceData, client *opsworks.Ops
 	d.SetId(layerId)
 	d.Set("id", layerId)
 
+	loadBalancer := aws.String(d.Get("elastic_load_balancer").(string))
+	if loadBalancer != nil && *loadBalancer != "" {
+		log.Printf("[DEBUG] Attaching load balancer: %s", *loadBalancer)
+		client.AttachElasticLoadBalancer(&opsworks.AttachElasticLoadBalancerInput{
```

Comment:
We should inspect the error here and respond accordingly. If it failed, we should return the error at least.